esrally/driver/runner.py

```diff
@@ -2032,6 +2037,55 @@ def __repr__(self, *args, **kwargs):
         return "wait-for-snapshot-create"
 
 
+class WaitForCurrentSnapshotsCreate(Runner):
+    """
+    Waits until all currently running snapshots on a given repository have completed
+    """
+
+    async def __call__(self, es, params):
+        repository = mandatory(params, "repository", repr(self))
+        wait_period = params.get("completion-recheck-wait-period", 1)
+        es_info = await es.info()
+        es_version = tuple([int(part) for part in es_info["version"]["number"].split(".")[:2]])
```

Comment:
Not a big deal, but you could perhaps use the [`versions.from_string()`](https://github.com/elastic/rally/blob/master/esrally/utils/versions.py#L98) method instead.

Reply:
Great catch. Fixed in 68951ec8ab0a845b73dff3aabc419713e317c10c